### PR TITLE
Fix ripples for checkboxes with `--paper-checkbox-size` in units other than 'px'.

### LINKED ISSUE #178: Ripple is incorrectly sized when `--paper-checkbox-size` is specified in units other than 'px'.
### Live Demo
http://jsbin.com/nevahus/5/edit?html,output

### PR DESCRIPTION
**Pull request by bicknellr**
Fixes #178.

### CODE REVIEW

File: paper-checkbox.html [hunk 1]
```diff
@@ -274,7 +274,17 @@
           var inkSize = this.getComputedStyleValue('--calculated-paper-checkbox-ink-size').trim();
           // If unset, compute and set the default `--paper-checkbox-ink-size`.
           if (inkSize === '-1px') {
-            var checkboxSize = parseFloat(this.getComputedStyleValue('--calculated-paper-checkbox-size').trim());
+            var checkboxSizeText = this.getComputedStyleValue('--calculated-paper-checkbox-size').trim();
+
+            // Is the value not in units of 'px'?
```
Comment (notwaldorf) [inline paper-checkbox.html]: I would explain here _why_ we need to do this, because it's really not obvious why we can't do the `calc` in all cases

Comment (notwaldorf) [inline paper-checkbox.html]: Update: you should just figure out what the unit is, do the same math, then add the same unit back. I think.

Comment (bicknellr) [inline paper-checkbox.html]: Fixed.

**Comment by notwaldorf** [comment]
LGTM! So good!!!

**Pull request merged**